packages/lwc/scripts/utils/rollup.js

```diff
@@ -5,46 +5,38 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 const path = require('path');
+
 const { rollup } = require('rollup');
-const typescript = require('typescript');
 const rollupTypescriptPlugin = require('rollup-plugin-typescript');
-const rollupReplace = require('@rollup/plugin-replace');
 const { terser: rollupTerser } = require('rollup-plugin-terser');
-const babel = require('@babel/core');
-const babelFeaturesPlugin = require('@lwc/features/src/babel-plugin');
-const { generateTargetName } = require('./helpers');
+const rollupReplace = require('@rollup/plugin-replace');
+const { default: nodeResolvePlugin } = require('@rollup/plugin-node-resolve');
 
-function rollupFeaturesPlugin(prod) {
-    return {
-        name: 'rollup-plugin-lwc-features',
-        transform(source) {
-            return babel.transform(source, {
-                plugins: [[babelFeaturesPlugin, { prod }]],
-            }).code;
-        },
-    };
-}
+const { generateTargetName } = require('./helpers');
 
 function rollupConfig(config) {
```

Comment:
The rollup build process should be refactored in a follow-up PR. As of today, rollup has to run multiple times to generate UMD, ESM, and IIFE. This is actually not needed since rollup can produce multiple outputs from the same bundle.